src/Host/Client/Test/DataInspection/ValuesTest.cs

```diff
@@ -18,6 +18,7 @@
 using Microsoft.R.Host.Client.Test.Script;
 using Microsoft.R.StackTracing;
 using Microsoft.UnitTests.Core.XUnit;
+using Microsoft.UnitTests.Core.XUnit.MethodFixtures;
```

Comment:
I am wondering if we should move away with strict naming by folders and simplify our namespaces a bit. Just a thought.